src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search__input {
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. Add only top or bottom, but not both to avoid potential margin collapse issues. In this case, you have margin-top on .search__input which might not be consistent with other elements.